framework/include/araui/ui_animation.h

```diff
@@ -42,14 +50,102 @@ typedef enum {
 extern "C" {
 #endif
 
+/**
+ * @brief Create a move animation
+ *
+ * If you want to get a current coordinate of widget, use ui_widget_get_rect.
+ *
+ * @param[in] from_x The x coordinate to move from
+ * @param[in] from_y The y coordinate to move from
+ * @param[in] to_x The x coordinate to move to
+ * @param[in] to_y The y coordinate to move to
+ * @param[in] duration Duration of animation
+ * @param[in] intrp_type Interpolation of animation
+ * @return On success, the handle of animation is returned. On failure, UI_NULL is returned.
+ *
+ * @see ui_widget_get_rect()
+ */
 ui_anim_t ui_move_anim_create(int32_t from_x, int32_t from_y, int32_t to_x, int32_t to_y, uint32_t duration, ui_intrp_type_t intrp_type);
+
+/**
+ * @brief Create a opacity animation
+ *
+ * @param[in] from_opacity The opacity to start
+ * @param[in] to_opacity The opacity to end
+ * @param[in] duration Duration of animation
+ * @param[in] intrp_type Interpolation of animation
+ * @return On success, the handle of animation is returned. On failure, UI_NULL is returned.
+ */
 ui_anim_t ui_opacity_anim_create(uint32_t from_opacity, uint32_t to_opacity, uint32_t duration, ui_intrp_type_t intrp_type);
+
+/**
+ * @brief Create a rotate animation
+ *
+ * @param[in] from_degree The degree to start
+ * @param[in] to_degree The degree to end
+ * @param[in] duration Duration of animation
+ * @param[in] intrp_type Interpolation of animation
+ * @return On success, the handle of animation is returned. On failure, UI_NULL is returned.
+ */
 ui_anim_t ui_rotate_anim_create(int32_t from_degree, int32_t to_degree, uint32_t duration, ui_intrp_type_t intrp_type);
+
+/**
+ * @brief Create a scale animation
+ *
+ * @param[in] from_scale The scale to start
+ * @param[in] to_scale The scale to end
+ * @param[in] duration Duration of animation
+ * @param[in] intrp_type Interpolation of animation
+ * @return On success, the handle of animation is returned. On failure, UI_NULL is returned.
+ */
 ui_anim_t ui_scale_anim_create(uint32_t from_scale, uint32_t to_scale, uint32_t duration, ui_intrp_type_t intrp_type);
+
+/**
+ * @brief Create a delay animation
+ *
+ * @param[in] duration Duration to delay
```

Comment:
How about change this parameter to `delay`. (e.g., `duration` -> `delay`)?

Reply:
@tyha1115 
I think you make sense.
But in perspective of unity, I think `duration` also makes sense.
Actually `ui_delay_anim_body_t` has no additional information except `ui_anim_body_t base`.
So... The `duration` of delay is set to the `ui_anim_body_t base` like `duration` of other animations.
How about you? @gcjjyy